tools/ide_plugins/vscode/package.json

```diff
@@ -34,7 +34,7 @@
     "cson": "^4.1.0",
     "tslint": "^5.8.0",
     "tslint-microsoft-contrib": "^5.0.1",
-    "typescript": "^2.6.1",
+    "typescript": "^3.7",
```

Comment:
I think it would be good to also add a lock file. This can make it easier to replicate successful builds. I can do this in a separate PR.